scripts/signature.py

```diff
@@ -93,10 +97,78 @@ def diff(args):
         """
         )
 
-    return exit_status
+    return delta, exit_status
+
+
+def diff(args):
+    """Diffs Velox function signatures."""
+    base_signatures = json.load(args.base)
+    contender_signatures = json.load(args.contender)
+    return diff_signatures(base_signatures, contender_signatures)[1]
+
+
+def dir_path(string):
+    if os.path.isdir(string):
+        return string
+    else:
+        raise NotADirectoryError(string)
 
 
-def parse_args():
+def bias(args):
+    base_signatures = json.load(args.base)
+    contender_signatures = json.load(args.contender)
+    output_dir = args.output_dir
+    tickets = args.ticket_value
+    bias_output, status = bias_signatures(
+        base_signatures, contender_signatures, tickets
+    )
+    function_bias_output = f"{output_dir}/bias_functions"
+    if status:
+        return status
+
+    if bias_output:
+        with open(function_bias_output, "w") as f:
+            print(f"{bias_output}", file=f, end="")
+
+    return 0
+
+
+def bias_signatures(base_signatures, contender_signatures, tickets):
+    """Returns newly added functions.
+    Contents are of the form : `fn_name1=10,fn_name2=10`.
```

Comment:
nit: could add that the 10` is the ticket value. Maybe even add 10 as a default value for ticket above?